src/cmds/Makefile.am

```diff
@@ -88,14 +88,19 @@ dist_bin_SCRIPTS = \
 
 common_cflags = \
 	-I$(top_srcdir)/src/include \
-	@KRB5_CFLAGS@
+	@KRB5_CFLAGS@ \
+	@libz_inc@
 
 common_libs = \
 	$(top_builddir)/src/lib/Libpbs/.libs/libpbs.a \
+	$(top_builddir)/src/lib/Libtpp/libtpp.a \
+	$(top_builddir)/src/lib/Liblog/liblog.a \
+	$(top_builddir)/src/lib/Libutil/libutil.a \
 	$(top_builddir)/src/lib/Libnet/libnet.a \
 	$(top_builddir)/src/lib/Libsec/libsec.a \
 	-lpthread \
 	@socket_lib@ \
+	@libz_lib@ \
```

Comment:
libz is required by TPP (aka libtpp.a), but earlier we had RPP funcs pointers (like pfn_rpp_*) so we didn't have to include this anywhere except where TPP was actually used (like pbs_comm, mom, server, etc...) because RPP funcs pointers was never set to TPP funcs in PBS client code.

But since after removal of RPP code (aka rpp.c and all __rpp_*() from it) and all RPP funcs pointers, now libpbs.a needs libtpp.a (due to use of DIS_tpp_funcs() in is_compose() in int_submit.c, which is part of libpbs.a)
And since we include libtpp.a, which needs libz (for compression) so these changes...
HTH!

Reply:
We can eliminate the need of tpp in cmds by defining dummy tpp fucntions in libcmds or similar place